src/python/grpcio_tests/tests/reflection/_reflection_servicer_test.py

```diff
@@ -40,8 +40,8 @@
 from google.protobuf import descriptor_pb2
 
 from src.proto.grpc.testing import empty_pb2
-
-from src.proto.grpc.testing.proto2 import empty2_pb2
+# empty2_pb2 is imported for import-consequent side-effects.
+from src.proto.grpc.testing.proto2 import empty2_pb2 # pylint: disable=unused-import
```

Comment:
Typographical minute nit-pick: [two spaces between code and beginning-of-comment octothorpe](https://google.github.io/styleguide/pyguide.html?showone=Whitespace#Whitespace).